tests/python/gaia-ui-tests/gaiatest/tests/brick/manifest.ini

```diff
@@ -0,0 +1,8 @@
+[parent:../manifest.ini]
+
```

Comment:
You'll still want to set external=false as the default is true, and this test doesn't require a network connection.